src/main/scala/org/hammerlab/guacamole/assembly/DeBruijnGraph.scala

```diff
@@ -123,13 +125,13 @@ class DeBruijnGraph(val kmerSize: Int,
   }
 
   /**
-   *  Searches forward or backward from a node to find those connected by a unique path
+    *  Searches forward or backward from a node to find those connected by a unique path
```

Comment:
sorry, I think I recently did the change this direction in at least one place, so that's probably my fault. happy to help with the munging if you like